util/config/config.go

```diff
@@ -120,7 +130,7 @@ func check(config *Configuration) error {
 			return errors.New("seed list in config file should not be blank")
 		}
 	default:
-		return fmt.Errorf("invalid consensus type %s in config file\n", config.ConsensusType)
+		return fmt.Errorf("invalid consensus type %s in config file", config.ConsensusType)
```

Comment:
Why get rid of '\n'?
fmt.Errorf will not supplement '\n' character automatically.

Reply:
My mistake. Ignored it